src/verifact_agents/claim_detector.py

```
@@ -1,3 +1,6 @@
+# Configure OpenAI client to use OpenRouter
+from src.utils.openrouter_config import *
```

Ruff (0.11.9)

`from src.utils.openrouter_config import *` used; unable to detect undefined names

(F403)